docs-internal/features/package-info.java

```diff
@@ -0,0 +1,18 @@
+@Feature(value = "Security/Providers/ABAC/Validators/EL",
```

Comment:
During the meeting we had this done on a module-info. Here it's done on a package-info, why ? Is it because this annotation is retained at runtime and we need a way to query the annotations regardless if the module-path is used or not ?

What's missing in this document is an example of how the features are queried for the startup list scenario. Also, for the `helidon features --list` this needs to be done statically without executing actual code, for this scenario it's easier to scan for resources files in the JAR files.

How about making this a compile time only annotation and generate a resource file during compilation with an annotation processor.

Also, I think the feature resource file should not be in a structured format that requires a library to be parsed. We need to be able to read such file without any additional library ; I guess a properties file is AOK.